editoast/src/views/infra/mod.rs

```diff
@@ -586,20 +597,57 @@ pub mod tests {
         let infra: Infra =
             call_and_read_body_json(&app, create_infra_request("get_voltages_test")).await;
 
+        let test_cases = vec![true, false];
```

Comment:
You can use `rstest`'s `case` annotation to be more idiomatic. https://github.com/la10736/rstest#parametrize

That would make the loop variable `include_rolling_stock_modes` below a function argument.